packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -132,7 +132,7 @@ abstract class Dart2WebTarget extends Target {
 
   @override
   List<Source> get outputs => buildFiles.map(
-    (String file) => Source.pattern('{OUTPUT_DIR}/$file')
+    (String file) => Source.pattern('{BUILD_DIR}/$file')
```

Comment:
Should there be a dart2web test that fails because of this change? are the `OUTPUT_DIR` and `BUILD_DIR` significantly different?

Reply:
I think the current test I added is basically sufficient for this, it ensures that the responsibilities for inputs and outputs in the `OUTPUT_DIR` and `BUILD_DIR` are reflected properly by the build system. I tried looking into doing an integration test, but it was difficult to actually build and pretty contrived, because you're essentially digging around and modifying things in the internal `.dart_tool/flutter_build` directory.